lib/medea/formatter.ex

```diff
@@ -36,9 +36,7 @@ defmodule Medea.Formatter do
   defp to_val(:message, message), do: Jason.encode_to_iodata!(message)
 
   defp to_val(:metadata, metadata) do
-    metadata
-    |> Utils.clean()
-    |> Jason.encode_to_iodata!()
+    Jason.encode_to_iodata!(Utils.clean([:metadata], metadata))
```

Comment:
Nitpick. Keep the piping so it matches `translator.ex`?

```suggestion
    [:metadata]
    |> Utils.clean(metadata)
    |> Jason.encode_to_iodata!()
```